apps/redi-connect/src/pages/front/login/Login.tsx

```diff
@@ -122,7 +122,9 @@ export default function Login() {
         return
       }
 
-      return history.push('/app/me')
+      const urlParams = new URLSearchParams(window.location.search)
+      const goto = urlParams.get('goto') ?? '/app/me'
+      return history.push(goto)
     } catch (err) {
       // Do nothing
```

Comment:
I know it's not in the scope of this ticket, but should we at least use "console.error" here?

Reply:
Good point, will do